vis-motions.c

```diff
@@ -16,10 +16,10 @@ static Regex *search_word(Vis *vis, Text *txt, size_t pos) {
 	if (!buf)
 		return NULL;
 	snprintf(expr, sizeof(expr), "[[:<:]]%s[[:>:]]", buf);
-	Regex *regex = vis_regex(vis, expr);
+	Regex *regex = vis_regex(vis, expr, strlen(expr));
 	if (!regex) {
 		snprintf(expr, sizeof(expr), "\\<%s\\>", buf);
-		regex = vis_regex(vis, expr);
+		regex = vis_regex(vis, expr, strlen(expr));
```

Comment:
These `strlen`s can be avoided by saving the return value from `snprintf`.

Reply:
I did that now by removing the ~500 character limit on the word search. That seemed to be better than handling the case where `snprintf` truncates its output.